src/components/issue-event-list-item.component.js

```diff
@@ -0,0 +1,617 @@
+    this.props.navigation.navigate('Profile', { user });
+  };
+
+  render() {
+    const { event } = this.props;
+
+    switch (event.event) {
+      case 'review_requested':
+        return (
+          <Event
+            icon={<EventIcon name="eye" />}
+            text={
+              <Text style={{ padding: 3 }}>
+                <ActorLink
+                  actor={event.review_requester}
+                  onPress={this.onPressUser}
+                />{' '}
+                requested review from{' '}
+                <ActorLink
+                  actor={event.requested_reviewer}
+                  onPress={this.onPressUser}
+                />
+              </Text>
+            }
+            createdAt={event.created_at}
+          />
+        );
+      case 'labeled':
+        return <Labeled event={event} onPressUser={this.onPressUser} />;
+      case 'unlabeled':
+        return (
+          <Labeled unlabeled event={event} onPressUser={this.onPressUser} />
+        );
+      case 'label-group':
+        return <LabelGroup group={event} onPressUser={this.onPressUser} />;
+      case 'closed':
+        return (
+          <Event
+            icon={
+              <EventIcon
+                name="circle-slash"
+                backgroundColor={colors.darkerRed}
+                iconColor={colors.white}
+              />
+            }
+            text={
+              <Text style={{ padding: 3 }}>
+                <ActorLink
+                  actor={event.actor}
+                  onPress={this.onPressUser}
+                />{' '}
+                closed this
+              </Text>
+            }
+            createdAt={event.created_at}
+          />
+        );
+      case 'reopened':
+        return (
+          <Event
+            icon={
+              <EventIcon
+                name="primitive-dot"
+                backgroundColor={colors.green}
+                iconColor={colors.white}
+              />
+            }
+            text={
+              <Text style={{ padding: 3 }}>
+                <ActorLink
+                  actor={event.actor}
+                  onPress={this.onPressUser}
+                />{' '}
+                reopened this
+              </Text>
+            }
+            createdAt={event.created_at}
+          />
+        );
+      case 'merged':
+        return (
+          <Event
+            icon={
+              <EventIcon
+                name="git-merge"
+                backgroundColor={colors.purple}
+                iconColor={colors.white}
+              />
+            }
+            text={
+              <Text style={{ padding: 3 }}>
+                <ActorLink
+                  actor={event.actor}
+                  onPress={this.onPressUser}
+                />{' '}
+                merged <Bold>{event.commit_id.slice(0, 7)}</Bold>
+              </Text>
+            }
+            createdAt={event.created_at}
+          />
+        );
+      // case 'referenced':
```

Comment:
Are the commented out cases conditions you just can't test? Noticing this when I view this issue #439 

![image](https://user-images.githubusercontent.com/12476932/31319805-7a047058-ac37-11e7-8c7f-50089439abb4.png)

Not sure why this is happening however :thinking:

Reply:
yeah, [the most recent commit](https://github.com/gitpoint/git-point/pull/438/commits/5f80eddf834f2a87c1e4392c3fa2b5838a1e5900#diff-291ee7c935dba219a987bfb7d7ff604bL194) should've removed that. i had that in for development purposes.